js/views/startScreen.js

```diff
@@ -25,6 +25,10 @@ var StartScreen = React.createClass({
     this.handleResize();
   },
 
+  componentWillReceiveProps: function() {
```

Comment:
@mistermendez I tried with this condition before creating pull request and it will not work. Reason is in handleResize(), this .setState.descriptionText is set based on this.props.contentTree.description and not based on nextProps.  So when ever there is a change in description it won't take affect.